### PR TITLE
feat: support policy evaluation

### PR DESCRIPTION
**Pull request by Gaiejj**
## Description

feat: support policy evaluation

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://omnisafe.readthedocs.io/en/latest/developer/contributing.html) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [x] I have updated the documentation accordingly.
- [x] I have reformatted the code using `make format`. (**required**)
- [x] I have checked the code using `make lint`. (**required**)
- [ ] I have ensured `make test` pass. (**required**)

### CODE REVIEW

File: omnisafe/common/normalizer.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # ==============================================================================
 """Implementation of Vector Buffer."""
```
Comment (zmsn-2077) [inline omnisafe/common/normalizer.py:15]: Vector Buffer should not appear in this file.

File: omnisafe/evaluator.py [hunk 2]
```diff
@@ -14,36 +14,290 @@
...
+
+        # set the render mode
+        self._play = play
+        self._save_replay = save_replay
+        self.__set_render_mode(play, save_replay)
+
+    def __set_render_mode(self, play: bool = True, save_replay: bool = True):
+        """Set the render mode.
 
         Args:
```
Comment (zmsn-2077) [inline omnisafe/evaluator.py:68]: I think the `Args` in the comment do not match the real function arguments.

File: omnisafe/evaluator.py [hunk 3]
```diff
@@ -14,36 +14,290 @@
...
 
-    def load_running_model(self, env, actor) -> None:
-        """Load running model from env and actor.
+    def __load_model_and_env(self, save_dir: str, model_name: str, env_kwargs: Dict[str, Any]):
+        """Load the model from the save directory.
 
         Args:
-            env (gym.Env): The environment.
-            actor (omnisafe.actor.Actor): The actor.
+            save_dir (str): directory where the model is saved.
```
Comment (zmsn-2077) [inline omnisafe/evaluator.py:101]: the same problems

File: omnisafe/evaluator.py [hunk 4]
```diff
@@ -14,36 +14,290 @@
...
+        """Evaluate the agent for num_episodes episodes.
 
         Args:
-            num_episode (int): The number of episodes to evaluate.
-            render (bool): Whether to render the environment.
+            num_episodes (int): number of episodes to evaluate the agent.
+            cost_criteria (float): the cost criteria for the evaluation.
 
+        Returns:
+            episode_rewards (list): list of episode rewards.
```
Comment (zmsn-2077) [inline omnisafe/evaluator.py:187]: need to fix,
        ```
        return (
            episode_rewards,
            episode_costs,
        )
        ```

**Review by zmsn-2077 — CHANGES_REQUESTED** [review]
.

**Review by rockmagma02 — APPROVED** [review]

**Review by zmsn-2077 — APPROVED** [review]
LGTM.

**Pull request merged**